### PR TITLE
Adding ability to return metrics for database sizes

### PR DESCRIPTION
**Pull request by naemono**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose

Returning database metrics

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [Unreleases]
+## [1.2.3]
```
Comment (majormoses) [inline CHANGELOG.md]: Please remove the version, we will version it when we release.

**Comment by majormoses** [comment]
Pulled this down and this looks good to me, just remove the versioning and I am good to merge and release this.

**Comment by naemono** [comment]
@majormoses requested changes complete.

**Comment by majormoses** [comment]
@naemono awesome ping me if I do not release within 24 hours.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-mongodb/versions/1.3.0

**Pull request merged**